### ISSUE TITLE
Some level db bugs break padplus store

### ISSUE THREAD

**Issue by su-chang** [wechaty/wechaty-puppet-padplus#201]
> Important：Please file the issue follow the template, or we won't help you to solve the problem.

## 0. Report Issue Guide

1. Please run the following command and check whether the problem has been fixed：

```
npm install wechaty-puppet-padlus@next
```

2. Please search in the issue first, and make sure your problem had not been reported before

## 1. Versions

- What is your wechaty version?
Answer: 0.35.5

- What is your wechaty-puppet-padplus version?
Answer: 0.7.2

- What is your node version? (run `node --version`)
Answer: 10.16.0

- What os are you using
Answer: Mac OS 10.15.4 

## 2. Describe the bug

Give a clear and concise description of what the bug is.
Can not get right data from flash-store, and the error stack shows

> ReadError: Unexpected end of JSON input

## 3. To Reproduce

This part is very important: if you can not provide any reproduce steps, then the problem will be very hard to be recognized.

1. run padplus sync data, set and get a lot of data

## 4. Expected behavior

Give a clear and concise description of what you expected to happen.

Flash-store could get data correctly.

## 5. Actual behavior

If applicable, add screenshots to help explain your problem. But do not paste log screenshots here.

Throw error when `Unexpected end of JSON input`

## 6. Full Output Logs

Set env `WECHATY_LOG=silly` in order to set log level to silly, then we can get the full log (If you dosen't set log env, log level is info as default, we cannot get the full log)

**We need full log instead of log screenshot or log fragments!**

<details>
<summary>
Show Logs
</summary>

```shell
$ WECHATY_LOG=silly node yourbot.js

Question: Paste your FULL(DO NOT ONLY PROVIDE FRAGMENTS) log messages
Answer:
ReadError: Unexpected end of JSON input
    at /xiaoju-bot/node_modules/levelup/lib/levelup.js:194:15
    at /xiaoju-bot/node_modules/encoding-down/index.js:79:14
    at /xiaoju-bot/node_modules/medeadown/medeadown.js:129:7
    at /xiaoju-bot/node_modules/medea/medea.js:551:15
    at FSReqWrap.wrapper [as oncomplete] (fs.js:463:17)
```

</details>

## 7. Additional context

[bug]

**Comment by windmemory** [comment]
We gonna rollback the `flash-store` version for now, and wait for `flash-store` to be updated and validated, then upgrade to the new version.

**Comment by su-chang** [comment]
This bug has already resolved in `wechaty-puppet-padplus@0.7.11`, and it could be closed.
